### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by designbymichael** [ekallevig/jShowOff#24]
Before starting with the issue, GREAT jquery script. Love it.

That said, there are a few accessibility issues you should address.  The hyperlinks shouldn't just be #null, but rather direct the user (particularly non-sighted users) to the anchor location of div they have selected.  From a sighted perspective, we are able to see a change in the presentation when we click on something, but when a non-sighted user activates a link (selects "next", "previous" or any of the other control links generated) it will change the content display, but they have no cue that a change has occurred and where it has occurred.

We'd love to use this within state government websites, but unfortunately because of the accessibility issue we cannot until this is addressed.  If you'd like to work on this together and come up with an accessible solution, contact me through github or gmail: michaeltangen.

**Comment by ekallevig** [comment]
Hey Michael,

This is a good point.  I have several enhancements I've been meaning to make, so I'll add this to the list and hopefully will have some time soon to work on it.  Let me know if there are other issues you noticed accessibility-wise.

Erik
